horovod/common/operations.cc

```diff
@@ -820,8 +820,9 @@ void PerformOperation(TensorTable& tensor_table, MPIResponse response) {
         // Copy memory into the fusion buffer.
         int64_t offset = 0;
         for (auto it = entries.begin(); it != entries.end(); it++) {
+          const void* buffer_data_at_offset = (uint8_t*)buffer_data + offset;
```

Comment:
Here, why don’t we directly define as a void *, but I’m ok with it.

Reply:
Sure, that's fine idea.